test/stress/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2019 Google LLC
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+FROM python:3.7.4
```

Comment:
Use https://github.com/GoogleContainerTools/distroless/blob/master/experimental/python3/README.md
The non-root flavor.

There's a PodSecurityPolicy that will disable running as root that I want to turn on.